scripts/download-realm.js

```diff
@@ -72,7 +72,23 @@ function printProgress(input, totalBytes, archive) {
     });
 }
 
+function copyLocal(archive, destination) {
+    const archivePath = `${process.env.HOME}/${archive}`;
```

Comment:
`archive` is randomized on each run, so `copyLocal` will only work if `REALM_DOWNLOAD_CORE_TEMP_DIR` environment variable is set (witch it is for testing I guess). But you could just use `REALM_DOWNLOAD_CORE_TEMP_DIR` instead, that might be a bit clearer.

Reply:
no this is for changing the temp dir. copyLocal is to use an pre downloaded binary to skip downloading it when testing.

Reply:
Ok.